tests/frontier/opcodes/test_calldataload.py

```diff
@@ -0,0 +1,116 @@
+"""test `CALLDATALOAD` opcode."""
+
+import pytest
+
+from ethereum_test_forks import Byzantium, Fork
+from ethereum_test_tools import Account, Alloc, Bytecode, StateTestFiller, Transaction
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+
+@pytest.mark.parametrize(
+    "mstore,args_size,code_for_address_a,tx_data,address_a_storage",
+    [
+        (
+            (Op.MSTORE8(offset=0x0, value=0x25) + Op.MSTORE8(offset=0x1, value=0x60)),
+            0x2,
+            (Op.PUSH1[0x0] + Op.CALLDATALOAD + Op.PUSH1[0x0] + Op.SSTORE),
+            b"\x00",
+            Account(
+                storage={0x00: 0x2560000000000000000000000000000000000000000000000000000000000000}
+            ),
```

Comment:
Could be simplified to:
```suggestion
            0x2560000000000000000000000000000000000000000000000000000000000000,
```
And then the function defines the `Account(storage=...` part.